gmp/gmp.py

```diff
@@ -218,7 +218,10 @@ def createFilterCommand(self, name, make_unique, kwargs):
 
         filter_type = kwargs.get('type', '')
         if filter_type:
-            if filter_type not in ('cc', 'snmp', 'up', 'usk'):
+            if filter_type not in ('Agent', 'Alert', 'Asset', 'Credential',
+             'Filter', 'Group', 'Note', 'Override', 'Permission', 'Port List',
+              'Report', 'Report Format', 'Result', 'Role', 'Schedule', 'SecInfo',
+               'Config', 'Tag', 'Target', 'Task', 'User'):
```

Comment:
Could you please put the tuple in a module (or class) variable? Also the indentation looks weird.